pkg/api/types.go

```diff
@@ -451,6 +451,8 @@ type ISCSIVolumeSource struct {
 	IQN string `json:"iqn,omitempty"`
 	// Required: iSCSI target lun number
 	Lun int `json:"lun,omitempty"`
+	// Optional: Defaults to 'default' (tcp). iSCSI interface name that uses an iSCSI transport.
+	ISCSIIface string `json:"iscsiIface,omitempty"`
```

Comment:
Keep in mind iscsiIface != Network interface. A standard name would be something like cxgb4i.00:07:43:28:b2:58 , do not want to confuse people into picking eth0